VERSION

```diff
@@ -1 +1 @@
-0.6.dev0
+0.6.dev1
```

Comment:
Can we change this only when putting forth a release?

Reply:
ok for now, but I see a need to exactly know what (GH commit-level) version is running, e.g. in case of issues.

Reply:
Let's cover in another ticket?

Reply:
Yes, another ticket, maybe even an "About" button.